app/models/ability.rb

```diff
@@ -52,6 +52,10 @@ def delivery_abilities
     end
   end
 
+  def designation_abilities
+    can [:index], Stockit::Designation if staff?
+  end
```

Comment:
@mcm-ham Would be good to add some tests for this in spec/models/abilities/designation_abilities.rb